api/pom.xml

```diff
@@ -131,7 +131,9 @@
                     <execution>
                         <goals>
                             <goal>set-spec-properties</goal>
+                            <!-- reenable before 2.0.0 release after plugin is fixed
```

Comment:
Can't plugin [2.1](https://github.com/eclipse-ee4j/glassfish-spec-version-maven-plugin/releases/tag/2.1) do? It has https://github.com/eclipse-ee4j/glassfish-spec-version-maven-plugin/pull/24 included.

Reply:
I didn't realize it had already been fixed, I'll update this to use the 2.1 version instead.